tests/test_dfa.py

```diff
@@ -977,90 +977,93 @@ def test_minify_dfa_complex(self):
                           '48', '22', '2', '36', '34', '49', '40', '25', '30',
                           '53', '42', '33', '19', '50', '29'})
         check_dfa = DFA(
-            states={'5', '36', '1', '49', '40', '25', '46', '6', '55',
-                    '{0,10,14,17,18,23,26,28,31,37,4,41,45,47,56,8}', '33',
-                    '11', '20', '48', '44', '32'},
+            states={frozenset(('5',)), frozenset(('36',)), frozenset(('1',)), frozenset(('49',)), frozenset(('40',)),
```

Comment:
@eliotwrobson In practice, will your algorithm changes still work even if developers opt to use non-frozen set literals instead of `frozenset`?

Reply:
What do you mean by "non-frozen set literals"? This is just asserting that the algorithm outputs frozensets for this test case.

Reply:
@eliotwrobson By non-frozen set literals, I was referring to the literal set syntax, e.g. `{1, 2, 3, ...}`. But I _just_ realized that wouldn't work anyway, since mutable sets are not hashable (i.e. you can't have a mutable set inside of another set).

So sorry about that—I was just overthinking it. 😅